frontend/src/components/Hero.tsx

```diff
@@ -8,14 +11,22 @@ const Hero = () => {
 
   return (
     <div data-testid="hero" className="usa-dark-background bg-primary">
-      <GridContainer className="padding-y-1 tablet:padding-y-3 tablet-lg:padding-y-10 desktop-lg:padding-y-15">
+      <GridContainer className="padding-y-1 tablet:padding-y-3 tablet-lg:padding-y-10 desktop-lg:padding-y-15 position-relative">
         <h1 className="tablet:font-sans-2xl desktop-lg:font-sans-3xl text-ls-neg-2">
           <span>{t("title")}</span>
         </h1>
         <p className="usa-intro font-sans-md tablet:font-sans-lg desktop-lg:font-sans-xl">
           <span className="text-yellow text-bold">{t("beta")}</span>&nbsp;
           {t("content")}
         </p>
+        <Link
+          className="usa-button usa-button--outline usa-button--inverse font-sans-2xs tablet:font-sans-md margin-bottom-3 desktop:position-absolute top-3 right-0 desktop:margin-y-3 desktop:margin-x-4"
+          href={ExternalRoutes.GITHUB_REPO}
+          target="_blank"
+        >
+          <Icon.Github className="usa-icon usa-icon--size-3 margin-right-1 text-middle tablet:margin-top-neg-2px" />
```

Comment:
Using the Truss `Icon` component gives this error: 

```
"<svg> elements with an img role must have an alternative text (svg-img-alt)"
```

…which was [reported in the Truss repo](https://github.com/trussworks/react-uswds/issues/2229), which indicates it's working as-intended. 

Not sure how to resolve this.

Reply:
@SammySteiner @daphnegold … halp pleeeze

Reply:
Oh, you just need to add `aria-label`. I'm pushing a fix upstream now.